compiler/resolution/visibleFunctions.cpp

```diff
@@ -954,7 +922,7 @@ BlockStmt* getVisibilityScope(Expr* expr) {
   while (cur != NULL) {
     // Pretend that ArgSymbols are in the function's body
     // (which is reasonable since functions cannot be defined
-    //  within an ArgSymbol).
+    //  within an ArgSymbol), so we consult POIs as needed.
```

Comment:
I'm not sure I understand what this addition means

Reply:
I meant to say that by returning `fn->body` in this case, we enable getVisibleFunctions() to visit the function's POI. Which is relied upon in the test mentioned in the next comment line, `test/functions/default-arguments/default-argument-generic.chpl`

Reply:
P.S. Let me know if you have a better way to phrase it.

Reply:
So I think what you're saying is that returning the function's body when getting the visibility scope for an ArgSymbol allows us to resolve things that ArgSymbol relies upon, if they are defined at the point of instantiation?  E.g.
```
module A {
  proc someFunc(type t, x = otherFunc()) { ... }
}
module B {
  use A;
  proc otherFunc() { ... }
  proc caller() {
    someFunc(int); // Where this is the point of instantiation, and that allows us to find B.otherFunc()?
  }
}
```

Reply:
If that's the case, I think I would rephrase this to: "Pretend that ArgSymbols are in the function's body (which is reasonable since functions cannot be defined within an ArgSymbol), so that we can consult the function's POIs when resolving the ArgSymbol."